framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -321,6 +328,24 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 			{ height: newBlockHeader.height },
 		);
 
+		const events = await this._sidechainAPIClient.invoke<JSONObject<chain.EventAttr[]>>(
+			'chain_getEvents',
+			{ height: newBlockHeader.height },
+		);
+
+		if (events) {
+			const ccmSendSuccessEvents = events.filter(
+				eventAttr =>
+					eventAttr.name === 'CcmSendSuccessEvent' &&
+					eventAttr.module === MODULE_NAME_INTEROPERABILITY,
```

Comment:
```suggestion
					eventAttr.name === 'ccmSendSuccess' &&
					eventAttr.module === MODULE_NAME_INTEROPERABILITY,
```
Also, maybe it's probably better to make it as a constant